use-my-tools/src/components/UsersTools/ToolsUserIsBorrowing.js

```diff
@@ -8,7 +8,9 @@ class ToolsUserIsBorrowing extends Component {
     tools: {}
   }
 
+
   render() {
+    console.log(localStorage.getItem("username"))
```

Comment:
This console.log is fine, but I would encourage you to remove it when this goes to production.